src/Markdig/Helpers/HtmlHelper.cs

```diff
@@ -430,7 +430,7 @@ private static bool TryParseHtmlTagHtmlComment(ref StringSlice text, ref ValueSt
 
         const string EndOfComment = "-->";
 
-        int endOfComment = slice.IndexOf(EndOfComment, StringComparison.Ordinal);
+        int endOfComment = slice.IndexOf(EndOfComment.AsSpan(), StringComparison.Ordinal);
```

Comment:
I included this change because for some reason, on macOS. `EndOfComment` wont compile. It gets:

> 0>HtmlHelper.cs(433,42): Error CS1503 : Argument 2: cannot convert from 'string' to 'System.ReadOnlySpan<char>'

So I've been adding `AsSpan()` but it may make sense to just get this in too? Can revert if you want.

Reply:
Was it failing for the Framework/standard builds?

Reply:
I don't know about framework as I'm on macOS, but it fails for net standard builds.